python/requirements_compiled.txt

```diff
@@ -2390,7 +2390,6 @@ transformers==4.19.1
     # via -r /ray/ci/../python/requirements/ml/core-requirements.txt
 trustme==0.9.0
     # via -r /ray/ci/../python/requirements/test-requirements.txt
-tune-sklearn @ git+https://github.com/ray-project/tune-sklearn@master
```

Comment:
why is this lost now?

Reply:
I was confused about this as well, I just took the file from the CI artifact: https://buildkite.com/ray-project/premerge/builds/9981#018b6dab-1668-4452-87e1-8d923270a410

Reply:
ah, it is removed.